bin/apn

```diff
@@ -126,6 +132,7 @@ command :feedback do |c|
 
   c.example 'description', 'apn feedback -c /path/to/apple_push_certificate.pem'
   c.option '-c', '--certificate CERTIFICATE', 'Path to certificate (.pem) file'
+  c.option '-r', '--rawcert CERTIFICATE VALUE', 'Value of the certificate (.pem) file'
```

Comment:
Same here.

Reply:
Perhaps we could add more description here? For example, how can one get value of the certificate or what would it look like.

Reply:
You mean something like `Actual content / value of the certificate (.pem) file`?

Reply:
Yes, something like that!